pyccel/parser/syntax/headers.py

```diff
@@ -157,6 +157,38 @@ def expr(self):
             d_var['order'] = order
         return d_var
 
+class MacroResult(BasicStmt):
+    """class representing a MacroResult in the grammar."""
+
+    def __init__(self, **kwargs):
+        """Constructor for MacroResult
+
+        name: str
+            Name of the variable result
+
+        shape: list
+            a list representing the shape of the result
+        """
+        self._name   = kwargs.pop('name')
+        self._shape  = kwargs.pop('shape', [])
+
+        super().__init__(**kwargs)
+
+    @property
+    def expr(self):
+        """."""
```

Comment:
This is not a docstring it is a pylint avoidance technique. Please use a real docstring

Reply:
Sorry, Fixed here ee14144